### ISSUE TITLE
pretty rtype logo ?

### ISSUE THREAD

**Issue by jokeyrhyme** [ericelliott/rtype#94]
I listened to this podcast recently:
- https://devchat.tv/js-jabber/205-jsj-shasta-with-eric-schoffstall

In it, @contra credits having a catchy logo as part of the success of his open-source projects. He even suggests something like https://www.squarespace.com/logo/ is a great way to get started with a logo.

Any ideas?

**Comment by Mouvedia** [comment]
I used to be logo designer.

**Comment by jokeyrhyme** [comment]
Type checking makes me think of shapes, and those children's toys with the blocks that have to be the right shape before they will fit:
- https://www.google.com.au/search?q=square+peg+round+hole&source=lnms&tbm=isch&sa=X
  
  /shrug

**Comment by ericelliott** [comment]
<img width="359" alt="screen shot 2016-04-17 at 11 16 49 pm" src="https://cloud.githubusercontent.com/assets/364727/14594327/87e5b72e-04f2-11e6-901b-caed9334953b.png">

**Comment by Mouvedia** [comment]
How plain.

Ill do one <ins>when Ill have time</ins>. It's been a while since Iv touched Illustrator.

**Comment by ericelliott** [comment]
😄

**Comment by jokeyrhyme** [comment]
We could also have a logo-treatment of some actual rtype text. We could use fun, frivolous examples of rtype for this:
- fn (rtype: Spec) => 😀

**Comment by maiermic** [comment]
That's easy :stuck_out_tongue_winking_eye: 

**JavaScript**
![JavaScript logo](https://geekoutsw.files.wordpress.com/2015/06/javascript-logo.png?w=102&h=102)

**TypeScript**
<img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" width="102">

**RType**
![rtype logo](http://i.stack.imgur.com/6P3xi.png)

**Comment by yocontra** [comment]
![image](https://cloud.githubusercontent.com/assets/425716/14648075/811048dc-062e-11e6-91d4-7bfa9f965eaf.png)

squarespace logo maker is indeed aight

**Comment by Mouvedia** [comment]
Looks like a logo for a natural language search engine.

**Comment by js-jslog** [comment]
How about these for a couple of concepts? Further derivations at http://js-jslog.github.io/
![Alt text](https://rlogo-jslog.c9users.io/js-jslog.github.io/rlogo_concept1.5.svg)
![Alt text](https://rlogo-jslog.c9users.io/js-jslog.github.io/rlogo_concept4.8.svg)

**Comment by Mouvedia** [comment]
What does the B/8/∞/tape/goggles/mask represent?

I also have a logo ready. I just don't have time right now to vectorize it.

**Comment by ericelliott** [comment]
@js-jslog Looks too much like a B, I think.

**Comment by js-jslog** [comment]
@ericelliott agreed. I've added a few more ideas [here](http://js-jslog.github.io/) (towards the bottom) which are more clearly R shaped. Out of interest, what inspired the name rtype? It might help stimulate logo ideas.

@Mouvedia The idea is that the 'type' is preceded by a colon (as you would find in the rtype syntax) and the R is built around that colon. Neither the R nor the colon were clear enough so I have tried to remedy that. Let me know what you think. Here's one example, there are a few more at http://js-jslog.github.io/.

If anyone wants to develop on this idea at all I would welcome that.

![Alt text](http://js-jslog.github.io/rlogo_concept4.13.svg)

**Comment by ericelliott** [comment]
[r-type](https://www.google.com/search?site=&tbm=isch&source=hp&biw=1920&bih=991&q=rtype&oq=rtype&gs_l=img.3..0l3j0i30l7.1899.2664.0.2857.5.4.0.1.1.0.131.399.2j2.4.0....0...1ac.1.64.img..0.5.400.5__q1HxCJwI) inspired the rtype name. =)

**Comment by js-jslog** [comment]
I'm pleased to hear that

**Comment by js-jslog** [comment]
![image](https://cloud.githubusercontent.com/assets/16046108/15470576/4ca6a6e2-20e9-11e6-92ae-6da7d9860fe3.png)
Solved.
